sim/core/attack.go

```diff
@@ -260,12 +263,76 @@ func (wa *WeaponAttack) setWeapon(weapon Weapon) {
 // inlineable stub for swing
 func (wa *WeaponAttack) trySwing(sim *Simulation) time.Duration {
 	if sim.CurrentTime < wa.swingAt {
+
+		if sim.Log != nil {
+			wa.unit.Log(sim, "trySwing failed - swing at %f and lastswing at %f", wa.swingAt, wa.lastSwingAt)
+		}
```

Comment:
Is this log just for debugging or is it used at all for parsing the timeline?

Reply:
just for debugging, I can remove it,